v3/src/components/case-table/collection-title.tsx

```diff
@@ -4,13 +4,20 @@ import { Editable, EditableInput, EditablePreview } from "@chakra-ui/react"
 import throttle from "lodash/throttle"
 import {useResizeDetector} from "react-resize-detector"
 import { observer } from "mobx-react-lite"
+import { clsx } from "clsx"
 import pluralize from "pluralize"
+import { uniqueName } from "../../utilities/js-utils"
 import { useDataSetContext } from "../../hooks/use-data-set-context"
 import { useCollectionContext } from "../../hooks/use-collection-context"
 import { getCollectionAttrs } from "../../models/data/data-set-utils"
 import t from "../../utilities/translation/translate"
+import AddIcon from "../../assets/icons/icon-add-circle.svg"
 
-export const CollectionTitle = observer(function CollectionTitle() {
+interface IProps {
+  isTileInFocus: boolean
```

Comment:
Rather than passing this in as a prop, it can be retrieved from `useCollectionModelContext()`.